sdk/python/feast/infra/offline_stores/contrib/spark_offline_store/spark.py

```diff
@@ -39,7 +39,7 @@
 
 
 class SparkOfflineStoreConfig(FeastConfigBaseModel):
-    type: StrictStr = "spark"
+    type: StrictStr = "spark.offline"
```

Comment:
This is going to be a backwards incompatible change for current users for the spark offline store, which breaks them. Why does it need to change?

Reply:
That is a good point, I'll revert.

Reply:
I have reverted this change